modelina-website/src/helpers/GeneratorCode/CSharpGenerator.ts

```diff
@@ -7,6 +7,10 @@ export function getCSharpGeneratorCode(
   const optionString: string[] = [];
   const optionStringPresets: string[] = [];
 
+  if (generatorOptions.csharpArrayType) {
+    optionString.push(`arrayType: '${generatorOptions.csharpArrayType}'`);
```

Comment:
The field name is different here - https://github.com/asyncapi/modelina/blob/513e04afc238b2b49b50cd0984d00bc3d85e2faf/src/generators/csharp/CSharpGenerator.ts#L35
So shouldn't we also specify 'collectionType' here instead of 'arrayType'?